README.md

```diff
@@ -29,17 +34,31 @@ This package provides various 'widgets' for `teal` applications. Examples includ
 
 ## Installation
 
-From July 2023 `insightsengineering` packages are available on [r-universe](https://r-universe.dev/).
-
 ```r
 # stable versions
-install.packages('teal.widgets', repos = c('https://insightsengineering.r-universe.dev', 'https://cloud.r-project.org'))
+install.packages('teal.widgets', repos = c('https://insightsengineering.r-universe.dev', getOption('repos')))
+
+# install.packages("pak")
+pak::pak("insightsengineering/teal.widgets@*release")
+```
+
+Alternatively, you might want to use the development version available on [r-universe](https://r-universe.dev/).
 
+```r
 # beta versions
-install.packages('teal.widgets', repos = c('https://pharmaverse.r-universe.dev', 'https://cloud.r-project.org'))
+install.packages('teal.widgets', repos = c('https://pharmaverse.r-universe.dev', getOption('repos')))
+
+# install.packages("pak")
+pak::pak("insightsengineering/teal.widgets")
 ```
 
-See package vignettes `browseVignettes(package = "teal.widgets")` for usage of this package.
+## Usage
+
+To understand how to use this package, please refer to the [package website](https://insightsengineering.github.io/teal.widgets/), which provides multiple examples of code usage.
```

Comment:
```suggestion
To understand how to use this package, please refer to the [package website](https://insightsengineering.github.io/teal.widgets/latest-tag/), which provides multiple examples of code usage.
```

Reply:
Also, just like what we did with teal.code and teal.logger, is it possible to add a few line of code to showcase the usage of teal.widgets?

Reply:
Hey @donyunardi `https://insightsengineering.github.io/teal.widgets/` automatically redirects to `https://insightsengineering.github.io/teal.widgets/latest-tag/` so no need for explicit usage of `latest-tag/` in the name. We put `latest-tag/` in urls that contains anything after `teal.widgets/` like `https://insightsengineering.github.io/teal.widgets/index.html` or ``https://insightsengineering.github.io/teal.widgets/articles/index.html` so it's `https://insightsengineering.github.io/teal.widgets/latest-tag/index.html` or ``https://insightsengineering.github.io/teal.widgets/latest-tag/articles/index.html`. But no need for regular website that ends with `teal.widgets`.

Reply:
> Also, just like what we did with teal.code and teal.logger, is it possible to add a few line of code to showcase the usage of teal.widgets?

@donyunardi the code for teal.widgets is way too big for quick view on README and I decided we will only point to the website for this package. I doesn't have any quick short snippets. It rather have massive code.

Reply:
Okay, that's fine.